pkg/workload/workload.go

```diff
@@ -96,6 +96,10 @@ func (i *Info) CanBePartiallyAdmitted() bool {
 	return false
 }
 
+func (i *Info) CanBePartiallyAdmitted() bool {
```

Comment:
worth keeping all methods belonging to Info together. So move this before `CanBePartiallyAdmitted`

Reply:
done